src/main/java/uk/gov/hmcts/ccd/data/definition/CachedUIDefinitionGateway.java

```diff
@@ -43,6 +43,13 @@ public SearchResult getSearchResult(final int version, final String caseTypeId)
         return httpUiDefinitionGateway.getSearchResult(version, caseTypeId);
     }
 
+    @Override
+    @Cacheable("searchCasesResultCache")
+    public SearchResult getSearchCasesResultDefinition(int version, String caseTypeId, String useCase) {
```

Comment:
minor: probably we could have considered removing the use case so we cache the whole SearchCasesResultFields tab to decrease the network calls when requesting different use cases